e2e/cypress/integration/notifications/reply_notifications_spec.js

```diff
@@ -201,20 +196,20 @@ describe('reply-notifications', () => {
 
             // # Wait till receiver's post is visible
             cy.getLastPostId().then(() => {
-                // # Switch to other channel so that unread notifications in 'town-square` may be triggered
-                cy.get(`#sidebarItem_${otherChannel.name}`).click({force: true});
+                // # Switch to other channel so that unread notifications in test channel may be triggered
+                cy.uiClickSidebarItem('town-square');
```

Comment:
What about a constant here also? But naming the naming is quite hard somehow :thinking: . `otherChannelName`, `switchChannelName`?

Reply:
Thanks for the feedback, and yeah, it's hard to name names 💯. I'll think more about the naming of variables since this is not specific to this case only. It would be nice if we could come up with a guide and make it as a semi-opinionated style.
For the meantime, I've updated the comment to "// # Switch to town-square...".